packages/swingset-runner/src/main.js

```diff
@@ -52,6 +52,7 @@ FLAGS may be:
   --lmdb           - runs using LMDB as the data store (default)
   --memdb          - runs using the non-persistent in-memory data store
   --dbdir DIR      - specify where the data store should go (default BASEDIR)
+  --dbsize SIZE    - set the LMDB size limit to SIZE megabytes (default 2GB)
```

Comment:
nit: SIZE is multiplied by `1024*1024`, not `1000*1000`, making the unit "mebibytes" instead of "megabytes" (and I think the default is similar, so it's really `2GiB` instead of `2GB`). I'm not certain that anyone else in the world is aware of the distinction, but I twitch a little when the units are ambiguous or wrong.

How well would it read if we changed it to `set the LMDB size limit to SIZE MiB (default 2GiB)`? If the user didn't know/care about the difference, would `MiB` seem similar enough to a familiar "MB" that they might get on with their morning and not fret about the unexpected extra vowel?

Reply:
I can never keep the abbreviations for decimal kilobytes and binary kilobytes straight (ditto mega, giga, etc).  My coping strategy is to always assume 1K is 1024 and pretend that 1000 is just a passing fad that I can ignore.